devito/builtins/initializers.py

```diff
@@ -2,11 +2,12 @@
 
 import devito as dv
 from devito.tools import as_tuple, as_list
-from devito.builtins.utils import nbl_to_padsize, pad_outhalo
+from devito.builtins.utils import abstract_args, nbl_to_padsize, pad_outhalo
```

Comment:
where is this used?

Reply:
I tried it here: https://github.com/devitocodes/devito/pull/2012/commits/65cd098196bec4e3e466c24693bcab334fb09698

then dropped it...

I was seeing a weird error in PRO, but to be fair it might just have been a confounder since I was fighting a batch of concurrent minor bugs

I left it here around because I think it's got potential and I hope to either get back to it soon or that someone will 😬